woocommerce/woocommerce
# dev: moved geolocation utils from wccom to here

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Migrating the geolocation utilities from wccom nux over to @woocommerce/onboarding as we will be using the same code for core profiler.

Follow up tasks:
- publish @woocommerce/onboarding
- import these utils from @woocommerce/onboarding in wccom and delete the duplicated code



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

No user facing changes.
Tests should run correctly, the only modifications were adding TS type definitions
